demo/site/src/common/blocks/StandaloneHeadingBlock.tsx

```diff
@@ -28,9 +25,9 @@ export const PageContentStandaloneHeadingBlock = (props: StandaloneHeadingBlockP
     </PageLayout>
 );
 
-const Root = styled.div<{ $textAlign: CSSProperties["textAlign"] }>`
-    text-align: ${({ $textAlign }) => $textAlign};
-`;
+const Root = styled.div<{ $textAlign: CSSProperties["textAlign"] }>({
+    textAlign: ({ $textAlign }) => $textAlign,
+});
```

Comment:
is the $ prefix in `$textAlign` needed? You removed it anywhere else. Would the prop then be forwarded automatically? (I guess the createShouldForwardPropList would be needed?)

Reply:
yeah true, there are no transient props (as far as i have seen) in pigment css. thats why i tried to remove them.

Reply:
You missed to remove the $ in the usage above.